Week-2/animation/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+#grandparent {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+#parent {
+  width: 200px;
+  height: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  flex-wrap: wrap;
+}
+
+.child-1,
+.child-2,
+.child-3,
+.child-4 {
+  height: 100px;
```

Comment:
You could've given them same class so you don't need to type like class-1, 2 etc.